central/imagecomponent/v2/datastore/store/postgres/gen.go

```diff
@@ -0,0 +1,3 @@
+package postgres
+
+//go:generate pg-table-bindings-wrapper --type=storage.ImageComponentV2 --references=storage.Image --search-category IMAGE_COMPONENTS_V2 --search-scope IMAGE_VULNERABILITIES_V2,IMAGE_COMPONENTS_V2,IMAGES,DEPLOYMENTS,NAMESPACES,CLUSTERS --feature-flag FlattenCVEData
```

Comment:
Do we need `IMAGE_COMPONENTS_V2` in `search-scope` too? It is already linked to the schema by `--search-category IMAGE_COMPONENTS_V2`

Reply:
well I simply copied the old ones and changed the value to V2.  So IDK and if so it would be a global change.  For example deployments references the deployment in both spots as well.  

So I'm reluctant to make that as part of this.  If you think we should investigate it (and we probably should), please file a ticket under https://issues.redhat.com/browse/ROX-28000

Reply:
Makes sense, I will create a ticket to track that.